httpbakery/client.go

```diff
@@ -313,6 +313,9 @@ func (c *Client) HandleError(reqURL *url.URL, err error) error {
 		return errgo.Notef(err, "cannot make cookie")
 	}
 	cookie.Path = cookiePath
+	if name := respErr.Info.CookieName; name != "" {
```

Comment:
I wondered about this earlier and thought that MacaroonName might be confusing because it's not really the name of the macaroon at all (and macaroons have an id which is really their name).

CookieNameSuffix?
MacaroonCookieName?
MacaroonCookieNameSuffix?